go.mod

```diff
@@ -1,9 +1,9 @@
-module github.com/DazWilkin/linode-exporter
+module main
```

Comment:
I didn't realize this approach is possible. Interesting, thanks!

It appears ancillary to the remainder of the PR and would be inconsistent with my other repos so, please revert.